### PR TITLE
Run acceptance tests against aks 1.19 nightly

### PR DESCRIPTION
**Pull request by ishustava**
Also, update `no_output_timeout` for all circle acceptance tests jobs to 1h just in case.

Passing build [here](https://app.circleci.com/pipelines/github/hashicorp/consul-helm/1831/workflows/8c1743f8-0a46-490e-ab4e-bd117d05b7ae/jobs/4687).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -128,7 +128,7 @@ jobs:
       - run:
           name: Run acceptance tests
           working_directory: test/acceptance/tests
-          no_output_timeout: 30m
+          no_output_timeout: 1h
```
Comment (kschoche) [inline .circleci/config.yml:131]: 👍🏽

**Review by lkysow — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -167,6 +167,77 @@ jobs:
...
+    steps:
+      - checkout
+
+      - run:
+          name: terraform init & apply
+          working_directory: test/terraform/aks
+          command: |
+            terraform init
+
+            terraform apply -var client_id="$ARM_CLIENT_ID" -var client_secret="$ARM_CLIENT_SECRET" -var cluster_count=2 -auto-approve
```
Comment (ndhanushkodi) [inline .circleci/config.yml:186]: Where are we setting $ARM_CLIENT_ID and $ARM_CLIENT_SECRET?

**Review by ndhanushkodi — APPROVED** [review]
🎉 This is awesome!

File: .circleci/config.yml (hunk 2 above)
Comment (ishustava) [reply .circleci/config.yml:186]: You can provide env variables to all jobs in circleci by setting them in [project settings](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project). If you go to our project settings for consul-helm, you should be able to see those.

**Pull request merged**